src/tags.rs

```diff
@@ -180,4 +183,13 @@ pub enum ResolutionUnit(u16) {
     Inch = 2,
     Centimeter = 3,
 }
+}
+
+tags! {
+pub enum SampleFormat(u16) {
```

Comment:
All the complex image TIFFs I've ever encountered just encode the imaginary component as an extra sample (or even extra image) rather than using this type. I also can't find reference to this outside of the mailing list thread from 2000 when it was added to libtiff. So far my policy with tag values has been to only include ones we can interpret. If we want to list values we can't interpret but know exist, we can change that policy. I can see the motivation if one wanted to make an `identify` or `tiffinfo` like tool using this library.

Reply:
Also should be clear it's not in the spec, not that "being in a 30-year-old spec" is a good criteria for what to include and what not to with a flexible format like TIFF.